packages/x-components/package.json

```diff
@@ -79,8 +79,8 @@
     "vuex": "^3.0.0"
   },
   "devDependencies": {
-    "@cypress/vue": "~2.2.4",
-    "@cypress/webpack-dev-server": "~1.8.4",
+    "@badeball/cypress-cucumber-preprocessor": "~13.0.1",
+    "@bahmutov/cypress-esbuild-preprocessor": "~2.1.4",
```

Comment:
cumpress-cucumber-preprocessor has been abandoned and [transferred to badeball](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689). Fortunately, there were just a few breaking changes. The `And` command does not exist anymore, the imports path has changed.